tests/pdifftests/test.py

```diff
@@ -26,15 +27,18 @@ class Runs(Base):
     window_size = [1280, 800]
     base_url = BASE
 
+    @hide('span.time')
```

Comment:
Ah, you also want to hide the time for `examine_comments-{edit,view}`, that's what I was seeing.